components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -78,6 +78,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
```

Comment:
Unnecessary new line

Reply:
This has already been fixed with https://github.com/wso2-extensions/identity-local-auth-basicauth/pull/142